### PR TITLE
Dev/lazuline

### PR DESCRIPTION
**Pull request by AbyssDweller**
#529 (remove like flames music disk and its .ogg ) completed
#528 (remove cooking pot) completed
#544 (high magisteel nugget) completed

### CODE REVIEW

**Review by Charismara — CHANGES_REQUESTED** [review]
please also remove the `like_flames` entry in the sounds.json file

File: src/main/java/com/github/manasmods/tensura/data/gen/TensuraRecipeProvider.java [hunk 1]
```diff
@@ -109,16 +109,16 @@ private void blocksAndItems(Consumer<FinishedRecipe> consumer) {
...
+                .requires(Items.ANVIL)
+                .requires(Items.SMITHING_TABLE)
+                .requires(Items.CRAFTING_TABLE)
+                .unlockedBy("has_anvil", inventoryTrigger(ItemPredicate.Builder.item()
+                        .of(Items.ANVIL).build()))
+                .unlockedBy("has_smithing_table", inventoryTrigger(ItemPredicate.Builder.item()
+                        .of(Items.SMITHING_TABLE).build()))
+                .unlockedBy("has_crafting_table", inventoryTrigger(ItemPredicate.Builder.item()
+                        .of(Items.CRAFTING_TABLE).build()))
+                .save(consumer);
```
[the remark this answers is not in the document]
Comment (AbyssDweller) [reply src/main/java/com/github/manasmods/tensura/data/gen/TensuraRecipeProvider.java:121]: its already at 4, idk why the indents are like that 
image

[the remark this answers is not in the document]
Comment (Charismara) [reply src/main/java/com/github/manasmods/tensura/data/gen/TensuraRecipeProvider.java:121]: hmm try changing the continuation indent to 4 too

**Comment by Charismara** [comment]
There are still some Cooking Pot things:

- [ ] TensuraBlockLoot.java:66
- [ ] TensuraBlocks.java:30
- [ ] TensuraBlockTagProvider.java:82
- [ ] TensuraBlockTagProvider.java:128

please remove them too

File: src/main/resources/assets/tensura/sounds.json [hunk 2]
```diff
@@ -2,8 +2,6 @@
   "like_flames": {
     "sounds": [
       {
-        "name": "tensura:like_flames",
-        "stream": true
       }
     ]
```
Comment (Charismara) [inline src/main/resources/assets/tensura/sounds.json:6]: Those lines need to be remove too. The remaining file is basically just the empty json object (due to the fact that we don't have any other sound yet)

Comment (AbyssDweller) [reply src/main/resources/assets/tensura/sounds.json:6]: yep, I forgot to push that one when I noticed that a couple minutes ago

**Review by Charismara — APPROVED** [review]
looks good to me. let's see that the pipeline says

**Pull request merged**